hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ClusterWideConfigurationService.java

```diff
@@ -84,7 +84,7 @@
         ConfigurationService,
         SplitBrainHandlerService {
 
-    public static final String SERVICE_NAME = "configuration-service";
+    public static final String SERVICE_NAME = ConfigurationService.SERVICE_NAME;
```

Comment:
minor: I think it would be best if we get rid of this completely.

Reply:
Done, EE PR  updated as well.